django_dicom/models/series.py

```diff
@@ -189,10 +189,11 @@ def get_data(self, as_json: bool = False) -> np.ndarray:
         """
 
         images = self.image_set.order_by("number")
-        data = np.stack([image.get_data() for image in images], axis=-1)
-        if as_json:
-            return json.dumps({"data": data}, cls=NumpyEncoder)
-        return data
+        # data = np.stack([image.get_data() for image in images], axis=-1)
+        # if as_json:
+        #     return json.dumps({"data": data}, cls=NumpyEncoder)
+        # return data
+        return np.stack([image.get_data() for image in images], axis=-1)
```

Comment:
Trying to return image data as JSON is ill-advised, therefore this option has been removed.